.github/workflows/build.yml

```
@@ -84,13 +84,15 @@ jobs:
             USER_GID=1000
 
       - name: Save PR ID
+        if: github.event_name == 'pull_request'
         env:
           PR_ID: ${{ github.event.number }}
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)